test/unit/uk/gov/hmrc/organisationsdetailsapi/services/ScopesConfig.scala

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.organisationsdetailsapi.services
+
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.Configuration
+import uk.gov.hmrc.organisationsdetailsapi.config.{ApiConfig, EndpointConfig, ScopeConfig}
+
+trait ScopesConfig extends MockitoSugar {
```

Comment:
It feels wrong to have this file in at this stage? Is this specific to employments?

Reply:
It is employments config but it's used through all the repos to simply check the handler pattern works as expected. We can update to be the same as whatever config gets generated but it's here to prove the functionlity/principle of the handler pattern.

Reply:
This is only because the specs are looking for employments. Which is wrong. Id rather this was just taken out for the time being rather than it being incorrect.